src/styles.scss

```diff
@@ -18,4 +18,6 @@ body {
 body {
     margin: 0;
     font-family: Roboto, "Helvetica Neue", sans-serif;
-}
\ No newline at end of file
+}
+/* Importing Bootstrap SCSS file. */
+@import '~bootstrap/scss/bootstrap';
```

Comment:
This is the old way of adding bootstrap to an angular project.
 `            "styles": [
              "./node_modules/@angular/material/prebuilt-themes/purple-green.css",
              "src/styles.scss",
              "node_modules/bootstrap/scss/bootstrap.scss"
            ],`
This is the newer way to do it in the angular.json file